django_forum/tests/functional_tests/features/posts/list_post/steps/test_list_post.py::py::test_user_lists_all_posts
--------------------------------------------------------------------
Last Page: http://localhost:43149/profile/
 Duration: 3.60s
  Browser: Chrome 109.0.5414.119 / headless2
   Driver: chromedriver 109.0.5414.74
Timestamp: 1677849370  (Unix Timestamp)
     Date: Friday, 3 March 2023
     Time: 1:16:10 PM  (UTC, UTC+00)
--------------------------------------------------------------------
Traceback:   File "/home/artisan/django_venv/lib/python3.11/site-packages/pytest_bdd/scenario.py", line 221, in scenario_wrapper
    _execute_scenario(feature, scenario, request)
  File "/home/artisan/django_venv/lib/python3.11/site-packages/pytest_bdd/scenario.py", line 193, in _execute_scenario
    _execute_step_function(request, scenario, step, step_func_context)
  File "/home/artisan/django_venv/lib/python3.11/site-packages/pytest_bdd/scenario.py", line 159, in _execute_step_function
    return_value = call_fixture_func(fixturefunc=context.step_func, request=request, kwargs=kwargs)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ceramic_isles_test/django_forum/tests/functional_tests/features/conftest.py", line 56, in post_exists
    return post()
           ^^^^^^
  File "/opt/ceramic_isles_test/django_forum/tests/functional_tests/features/conftest.py", line 62, in posty
    return forum_models.Post.objects.get_or_create(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/query.py", line 936, in get_or_create
    return self.create(**params), True
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/query.py", line 671, in create
    obj.save(force_insert=True, using=self.db)
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django_messages/models.py", line 59, in save
    super().save(force_insert, force_update, using, update_fields)
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/base.py", line 812, in save
    self.save_base(
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/base.py", line 863, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/base.py", line 1006, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/base.py", line 1047, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/query.py", line 1791, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1659, in execute_sql
    for sql, params in self.as_sql():
                       ^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1583, in as_sql
    value_rows = [
                 ^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1584, in <listcomp>
    [
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1585, in <listcomp>
    self.prepare_value(field, self.pre_save_val(field, obj))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1533, in pre_save_val
    return field.pre_save(obj, add=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/fields/__init__.py", line 905, in pre_save
    return getattr(model_instance, self.attname)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/query_utils.py", line 182, in __get__
    instance.refresh_from_db(fields=[field_name])
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/base.py", line 722, in refresh_from_db
    db_instance = db_instance_qs.get()
                  ^^^^^^^^^^^^^^^^^^^^
  File "/home/artisan/django_venv/lib/python3.11/site-packages/django/db/models/query.py", line 650, in get
    raise self.model.DoesNotExist(

Exception: %sPost matching query does not exist.